cla_frontend/settings/base.py

```diff
@@ -122,6 +122,7 @@ def get_env_value(var_name):
     "wss:",
     "www.google-analytics.com",
     "stats.g.doubleclick.net",
+    "'unsafe-inline'",
```

Comment:
Remind me what this was fixing and, specifically, was it an issue on staging/training/prod or just local dev?

Reply:
This was fixing executing code inside a `script` tag. Was an issue on local dev.

We don't have any code that executes inside a script tag, so this issue wouldn't be present on any env.

Reply:
OK. If it's only on local, can you think of a better place to put this exemption?
It's a security feature, so we don't want to disable it where we don't have to, even if it won't have any effect right now.

Reply:
Yeah this should go in `local.py`

I am not sure if this will work on staging/prod/training or not since we haven't tested it/don't have any JS code that runs in a script. 

I will put it in `local.py` and see if it works on staging.

Reply:
Nah there's an even easier solution. Have a little look around this file, it's really nearby!

> I am not sure if this will work on staging/prod/training or not since we haven't tested it/don't have any JS code that runs in a script.

Not sure what'll work!?

Reply:
So the CSP issue which required me writing this line of code flagged up my on local because I was working locally. There are no script tags that run JS code in our codebase, so idk if we need this line on staging/training/prod.

Reply:
Was this more of what you was looking for?
`if "localhost" in ALLOWED_HOSTS:
    CSP_DEFAULT_SRC += ["localhost:*", "'unsafe-inline'"]`

Reply:
Yes that's spot on.

And yes if we don't need it on non-local then we shouldn't have it, but obviously you need to test it, cos "There are no script tags that run JS code in our codebase", well, haven't we just added one?

Reply:
Yay

Yeah I guess the fact we would need to test it was going to be obvious if we are trying to see whether we need it or not. TDD starting to feel like a lifestyle at this point 😅 (jk)

Reply:
Done the changes. Will test on staging now

Reply:
Tested on staging. We get the same error. Reverted the last commit

Reply:
Right, I thought that might be the case.